sdk/servicebus/Azure.Messaging.ServiceBus/tests/Message/MessageLiveTests.cs

```diff
@@ -222,6 +224,7 @@ await using (var scope = await ServiceBusScope.CreateWithTopic(enablePartitionin
                 Assert.IsFalse(rawSend.MessageAnnotations.ContainsKey(AmqpMessageConstants.EnqueueSequenceNumberName));
                 Assert.IsFalse(rawSend.MessageAnnotations.ContainsKey(AmqpMessageConstants.EnqueuedTimeUtcName));
                 Assert.IsFalse(rawSend.MessageAnnotations.ContainsKey(AmqpMessageConstants.DeadLetterSourceName));
+                Assert.IsFalse(rawSend.MessageAnnotations.ContainsKey(AmqpMessageConstants.MessageStateName));
```

Comment:
Can we update existing a few Deferral/Schedule/regular message tests to check the property?

Reply:
Sure will do. Good catch

Reply:
Pushed. Let me know if you can think of some more test candidates